pkg/api/postage.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+import (
+	"encoding/hex"
+	"math/big"
+	"net/http"
+	"strconv"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/gorilla/mux"
+)
+
+func (s *server) postageCreateHandler(w http.ResponseWriter, r *http.Request) {
+	depthStr := mux.Vars(r)["depth"]
+
+	amount, ok := big.NewInt(0).SetString(mux.Vars(r)["amount"], 10)
+	if !ok {
+		jsonhttp.BadRequest(w, "invalid postage amount")
```

Comment:
it should be:
```go
s.Logger.Debugf("create batch: parse amount: %v",err)
s.Logger.Error("create batch: invalid amount")
jsonhttp.BadRequest(w, "invalid postage amount")
return
```

Reply:
what if there is no error like in this case. should there still be two log lines?

Reply:
sorry, yeah in this case just an error would probably do :+1: